packages/paypal-commerce-integration/src/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -91,16 +107,25 @@ export default class PayPalCommercePaymentStrategy implements PaymentStrategy {
             throw new PaymentArgumentInvalidError(['payment']);
         }
 
+        const { methodId, paymentData } = payment;
+
+        if (this.isPayPalVaultedInstrumentPaymentData(paymentData)) {
+            if (!this.orderId) {
```

Comment:
We can combine this 2 checks into one 
```suggestion
        if (this.isPayPalVaultedInstrumentPaymentData(paymentData) && !this.orderId) {

```